desc/compute/_basis_vectors.py

```diff
@@ -1400,10 +1400,39 @@ def _e_sup_zeta_zz(params, transforms, profiles, data, **kwargs):
         "desc.equilibrium.equilibrium.Equilibrium",
         "desc.geometry.surface.FourierRZToroidalSurface",
     ],
+    aliases=["e_phi"],
+    # Our usual notation implies e_phi = (∂X/∂Φ)|R,Z, but we need to alias e_phi to
```

Comment:
I thought our usual assumption would be rho, theta held constant? Where do we need derivatives with R,Z constant?

Reply:
It's more like we hold the variables in the natural coordinate system constant.

For example, `e_theta` is at fixed rho and zeta because the coordinate system is (rho, theta, zeta).
Likewise, `e_theta_PEST = e_vartheta` is at fixed rho and phi because the coordinate system is (rho, vartheta, phi).
I have followed this convention in the PRs that add quantities derivatives along field lines such as
`e_alpha` which is at fixed rho and zeta because the coordinate system is (rho, alpha, zeta).

Same for `B_z` is at fixed rho, theta, but `B_a` is at fixed rho, zeta.

Reply:
This is moreso to clarify the inconsistency that `e_phi` is not `e_phi|R,Z`. One could have also used this when computing `A(z)`.